app/models/explainer.rb

```diff
@@ -107,7 +107,8 @@ def self.update_paragraphs_in_alegre(id, previous_paragraphs_count, timestamp)
     end
   end
 
-  def self.search_by_similarity(text, language, team_id)
+  def self.search_by_similarity(text, language, team_id, limit = nil)
+    limit ||= CheckConfig.get(:most_relevant_team_limit, 3, :integer)
```

Comment:
I suggest we always enforce a `limit` to be passed (see other comments).

Reply:
Done